saos-persistence/src/main/java/pl/edu/icm/saos/persistence/model/RemovedJudgment.java

```diff
@@ -0,0 +1,56 @@
+package pl.edu.icm.saos.persistence.model;
+
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+
+import pl.edu.icm.saos.persistence.common.DataObject;
+
+/**
+ * Entity representing judgment that have been
+ * removed from database
+ * 
+ * @author madryk
+ */
+@Entity
+@SequenceGenerator(name = "seq_removed_judgment", allocationSize = 1, sequenceName = "seq_removed_judgment")
+public class RemovedJudgment extends DataObject {
```

Comment:
Well, I would change it to DeletedJudgment because it can be deleted for many reasons in the future.